src/clib/lib/include/ert/enkf/enkf_config_node.hpp

```diff
@@ -126,10 +118,10 @@ bool enkf_config_node_internalize(const enkf_config_node_type *node,
 /*
     The enkf_node_free() function declaration is in the enkf_config_node.h header,
     because the enkf_config_node needs to know how to free the min_std node.
+    That's not really the reason, because min_std doesn't exist anymore, but
```

Comment:
This comment is a bit unhelpful. But I don't really have a better suggestion than to move the function in the `enfk_node.h` file, which I am not sure it should be done as part of this PR. So I guess leave it as it is :man_shrugging:

Reply:
thanks for looking at / into it!